site/content/docs/features/config-file.md

```diff
@@ -1 +1,632 @@
+
+_**HealthCheckTimeout**_  
+The time, in seconds, to wait for the response before deciding that a health check failed. The valid range of values if 1 through 20, and the default is 2.
+
+_**HealthCheckHealthyThreshold**_  
+The number of consecutive checks that must succeed before AWS App Runner decides that the service is healthy. The valid range of values if 1 through 20, and the default is 3.
+
+_**HealthCheckUnhealthyThreshold**_  
+The number of consecutive checks that must fail before AWS App Runner decides that the service is unhealthy. The valid range of values if 1 through 20, and the default is 3.
+
+_**VPCConnector**_  
+The VPC connector that associates your App Runner service with a custom Amazon VPC. For more information about VPC connectors, see [Networking with App Runner](https://docs.aws.amazon.com/apprunner/latest/dg/network.html) in the [AWS App Runner Developer Guide](https://docs.aws.amazon.com/apprunner/latest/dg/what-is-apprunner.html).
+
+By default, the deployment tool will not create a new VPC connector, but expects the ID of an existing VPC connector in `VpcConnectorId`.  
+To create a new VPC connector, set `CreateNew` to `true` and provide the following information:
+
+* `VpcId`  
+The VPC ID that AWS App Runner should use when it associates your service with a custom Amazon VPC.
+* `Subnets`  
+A list of subnet IDs that AWS App Runner should use when it associates your service with a custom Amazon VPC. Specify IDs of subnets of a single Amazon VPC. AWS App Runner determines the Amazon VPC from the subnets you specify.
+* `SecurityGroups`  
+A list of IDs of security groups that AWS App Runner should use for access to AWS resources under the specified subnets. If not specified, AWS App Runner uses the default security group of the custom Amazon VPC. The default security group allows all outbound traffic.
+
+_**AppRunnerEnvironmentVariables**_  
+Environment properties for your application.
+
+## JSON syntax for Docker container image to Amazon ECR
+
+Use this JSON definition to push a Docker container image to [Amazon Elastic Container Registry (Amazon ECR)](https://docs.aws.amazon.com/ecr) (`RecipeId` set to "PushContainerImageEcr").
+
+```
+{
+    "AWSProfile": "string",
+    "AWSRegion": "string",
+    "StackName": "string",
+    "RecipeId": "PushContainerImageEcr",
+    "OptionSettingsConfig":
+    {
+        "ImageTag": "string",
+    }
+}
+```
+
+The following parameter definitions are specific to the JSON syntax for pushing a Docker container image to Amazon ECR. Also see [Common JSON parameters](#common-json-parameters).
+
+_**ImageTag**_  
+The tag associated with the container images that are pushed to Amazon ECR.  
+The image tag can contain only the following characters: uppercase and lowercase letters, digits, dashes, periods (.) and underscores (_). It can contain a maximum of 128 characters and cannot start with a special character.
```

Comment:
The default value is "{DefaultECRImageTag}".
What does that mean?
Is it "latest", as mentioned in the ECR dev guide?